examples/todo-mvc/ui/widgets/selection_list_view.slint

```diff
@@ -48,6 +48,7 @@ export component SelectionListViewItemDelegate {
         check-box := CheckBox {
             horizontal-stretch: 0;
             y: (parent.height - self.height) / 2;
+            accessible-label: @tr("selected");
```

Comment:
This checkbox indicates whether a task is done right? "selected" is misleading.
The proper solution here would be to build the checkbox name from the task name and the due date: "<task name>, due by <due date>"